packages/front-end/components/Stepper/Stepper.tsx

```diff
@@ -0,0 +1,79 @@
+import clsx from "clsx";
+import { MdCheck } from "react-icons/md";
+import { PiCircleDashed } from "react-icons/pi";
+
+type Step = {
+  label: string;
+  enabled: boolean;
+};
+
+interface Props {
+  step: number;
+  setStep: (step: number) => void;
+  steps: Step[];
+  setError: (error: string) => void;
+  skipped?: Set<number>;
+  validateSteps?: (step: number) => Promise<void>;
+}
+
+export default function Stepper({
+  step,
+  setStep,
+  steps,
+  skipped,
+  setError,
+  validateSteps,
+}: Props) {
+  return (
+    <nav
+      className={
+        "nav mb-4 justify-content-start nav-default paged-modal-default"
```

Comment:
can we convert the spacing to radix spacing most of this is using bootstrap

Reply:
Yeah I can, but this isn't a radix component so I don't think I can do it on the component itself. Is what you had in mind using the CSS variables they have available within the CSS for this?